deps/v8/src/heap/cppgc/heap-base.h

```diff
@@ -331,7 +331,7 @@ class V8_EXPORT_PRIVATE HeapBase : public cppgc::HeapHandle {
   friend class cppgc::testing::OverrideEmbedderStackStateScope;
 };
 
-class V8_NODISCARD V8_EXPORT_PRIVATE ClassNameAsHeapObjectNameScope final {
+class ClassNameAsHeapObjectNameScope final {
```

Comment:
Given that Chromium is built with clang-cl (and clang-cl is required), why would v8 not build with clang-cl out of the box?

Reply:
No longer necessary.